src/server/game/Handlers/TicketHandler.cpp

```diff
@@ -111,6 +112,31 @@ void WorldSession::HandleGMTicketCreateOpcode(WorldPacket& recvData)
         sWorld->SendGMText(LANG_COMMAND_TICKETNEW, GetPlayer()->GetName().c_str(), ticket->GetId());
 
         response = GMTICKET_RESPONSE_CREATE_SUCCESS;
+
+        if ((sIRC->BOTMASK & 1024) != 0 && (sIRC->TICMASK & 1) != 0 && sIRC->ticann.size() > 0)
+        {
+            std::string ircchan = "#";
+            std::ostringstream smsg;
+            ircchan += sIRC->ticann;
+            smsg << "[By: \00304" << GetPlayer()->GetName().c_str() << "\003 ][ID: \00304" << ticket->GetId() << "\003 ]";
+            sIRC->Send_IRC_Channel(ircchan, sIRC->MakeMsg("[\00304Ticket Created\003] %s", " %s", smsg.str().c_str()), true);
+            //begin // because newlines don't show on irc
+            std::string tmsg = ticket->GetMessage().c_str();
+            int position = tmsg.find( "\n" ); // find first newline
+            while ( position != int(string::npos) )
+            {
+                tmsg.replace( position, 1, "|" );
+                position = tmsg.find( "\n", position + 1 );
+            }
+            //end
+            if (tmsg.size() <= 400)
+                sIRC->Send_IRC_Channel(ircchan, sIRC->MakeMsg("[\00304Ticket Message\003]: %s ", " %s", tmsg.c_str()), true);
+            else
+            {
+                sIRC->Send_IRC_Channel(ircchan, sIRC->MakeMsg("[\00304Ticket Message(1/2)\003]: %s ", " %s", tmsg.substr(0, 399).c_str()), true);
+                sIRC->Send_IRC_Channel(ircchan, sIRC->MakeMsg("[\00304Ticket Message(2/2)\003]: %s ", " %s", tmsg.substr(399, 800).c_str()), true);
```

Comment:
No but 512 should be the max since its IRC's message limit

IRC messages are always lines of characters terminated with a CR-LF (Carriage Return - Line Feed) pair, and these messages SHALL NOT exceed 512 characters in length, counting all characters including the trailing CR-LF. Thus, there are 510 characters maximum allowed for the command and its parameters.